### ISSUE TITLE
Scanned items counting by kind in report is incorrect (with exclusions)

### ISSUE THREAD

**Issue by Berber31** [derailed/popeye#210]
<img src="https://raw.githubusercontent.com/derailed/popeye/master/assets/popeye_bug.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Describe the bug**
Scanned Items counting by kind in the report is incorrect when we exclude some codes.

**To Reproduce**
1. Deploy 1 pod in `my-ns` namespace and declare exclusions:
  _config.yaml_
  ```yaml
  popeye:
    excludes:
      v1/pods:
        - name: rx:.*
          codes:
            - 105
            - 108
            - 109
            - 110
            - 111
            - 112
            - 201
            - 202
            - 203
            - 204
            - 205
            - 206
            - 207
            - 300
            - 301
            - 303
  ```

2. run `popeye -s pod -n my-ns -f ./config.yaml`
3. Inspect report in pods section
4. Pods section report `PODS 0 SCANNED`
  ```
  PODS (0 SCANNED)                                                             💥 0 😱 0 🔊 0 ✅ 0 100٪
  ┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅
    · Nothing to report.
  ```

**Expected behavior**
Scan report with:
  ```
  PODS (1 SCANNED)                                                             💥 0 😱 0 🔊 0 ✅ 1 100٪
  ┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅┅
    · Nothing to report.
  ```

**Screenshots**
NA

**Versions (please complete the following information):**
 - OS: ubuntu
 - Popeye: 0.9.8
 - K8s: 1.20

**Additional context**
NA

**Comment by pbaye** [comment]
i notice the same behaviour, but replacing v1/pods by v1/Pods in the spinach.yaml file, now i see me pods in the popeye report.
i have the same case-sensitive effect on other kinds

**Comment by derailed** [comment]
Fixed v0.20.0
